mlrun/utils/http.py

```diff
@@ -106,10 +107,7 @@ def request(self, method, url, **kwargs):
 
                 # only retry on exceptions with the right message
                 exception_is_retryable = any(
-                    [
-                        msg in str(exc)
-                        for msg in self.HTTP_RETRYABLE_EXCEPTIONS[type(exc)]
-                    ]
+                    [msg in str(exc) for msg in self.HTTP_RETRYABLE_EXCEPTION_STRINGS]
```

Comment:
```suggestion
 msg in str(exc) for msg in self.HTTP_RETRYABLE_EXCEPTION_STRINGS
```

no need to preallocate a list